### ISSUE TITLE
Valet use 7.3 Breaks valet 

### ISSUE THREAD

**Issue by kupoback** [weprovide/valet-plus#320]
Typing in `valet use 7.3` has broken valet entirely.

This is the error I get, and I cannot stop and restart valet:

```
valet use 7.3
[php@7.2] Unlinking
Unlinking /usr/local/Cellar/valet-php@7.2/7.2.13... 7 symlinks removed

[libjpeg] Relinking
[php@7.3] Linking
Linking /usr/local/Cellar/valet-php@7.3/7.3.1... 148 symlinks created

If you need to have this software first in your PATH instead consider running:
  echo 'export PATH="/usr/local/opt/valet-php@7.3/bin:$PATH"' >> ~/.bash_profile
  echo 'export PATH="/usr/local/opt/valet-php@7.3/sbin:$PATH"' >> ~/.bash_profile

[valet-php@7.1] Stopping
[valet-php@7.2] Stopping
[valet-php@7.3] Stopping
[BREW TAP] henkrehorst/php already installed
[PECL] Updating PECL channel: pecl.php.net
[PECL] Installing extensions
PHP Warning:  file_get_contents(PHP Warning:  Module 'apcu' already loaded in Unknown on line 0Warning: Module 'apcu' already loaded in Unknown on line 0/usr/local/etc/valet-php/7.3/php.ini): failed to open stream: No such file or directory in /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Filesystem.php on line 112
PHP Stack trace:
PHP   1. {main}() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:0
PHP   2. Silly\Application->run() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:922
PHP   3. Silly\Application->doRun() /Users/nmak/.composer/vendor/symfony/console/Application.php:148
PHP   4. Silly\Application->doRunCommand() /Users/nmak/.composer/vendor/symfony/console/Application.php:248
PHP   5. Silly\Command\Command->run() /Users/nmak/.composer/vendor/symfony/console/Application.php:953
PHP   6. Silly\Application->Silly\{closure}() /Users/nmak/.composer/vendor/symfony/console/Command/Command.php:253
PHP   7. Invoker\Invoker->call() /Users/nmak/.composer/vendor/mnapoli/silly/src/Application.php:85
PHP   8. call_user_func_array:{/Users/nmak/.composer/vendor/php-di/invoker/src/Invoker.php:82}() /Users/nmak/.composer/vendor/php-di/invoker/src/Invoker.php:82
PHP   9. Silly\Application->{closure:/Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:484-486}() /Users/nmak/.composer/vendor/php-di/invoker/src/Invoker.php:82
PHP  10. Facade::switchTo() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:485
PHP  11. Facade::__callStatic() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:485
PHP  12. Valet\PhpFpm->switchTo() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/includes/facades.php:28
PHP  13. Valet\PhpFpm->install() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/PhpFpm.php:178
PHP  14. Valet\Pecl->installExtensions() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/PhpFpm.php:75
PHP  15. Valet\Pecl->installExtension() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php:92
PHP  16. Valet\Pecl->install() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php:113
PHP  17. Valet\Filesystem->get() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php:142
PHP  18. file_get_contents() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Filesystem.php:112

Warning: file_get_contents(PHP Warning:  Module 'apcu' already loaded in Unknown on line 0Warning: Module 'apcu' already loaded in Unknown on line 0/usr/local/etc/valet-php/7.3/php.ini): failed to open stream: No such file or directory in /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Filesystem.php on line 112

Call Stack:
    0.0065     391744   1. {main}() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:0
    0.0766     970904   2. Silly\Application->run() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:922
    0.0942    1072448   3. Silly\Application->doRun() /Users/nmak/.composer/vendor/symfony/console/Application.php:148
    0.0947    1072448   4. Silly\Application->doRunCommand() /Users/nmak/.composer/vendor/symfony/console/Application.php:248
    0.0951    1072448   5. Silly\Command\Command->run() /Users/nmak/.composer/vendor/symfony/console/Application.php:953
    0.0956    1077656   6. Silly\Application->Silly\{closure}() /Users/nmak/.composer/vendor/symfony/console/Command/Command.php:253
    0.0956    1078688   7. Invoker\Invoker->call() /Users/nmak/.composer/vendor/mnapoli/silly/src/Application.php:85
    0.0957    1079256   8. call_user_func_array:{/Users/nmak/.composer/vendor/php-di/invoker/src/Invoker.php:82}() /Users/nmak/.composer/vendor/php-di/invoker/src/Invoker.php:82
    0.0958    1079648   9. Silly\Application->{closure:/Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:484-486}() /Users/nmak/.composer/vendor/php-di/invoker/src/Invoker.php:82
    0.0958    1079648  10. Facade::switchTo() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:485
    0.0958    1080024  11. Facade::__callStatic() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/valet.php:485
    0.1027    1161344  12. Valet\PhpFpm->switchTo() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/includes/facades.php:28
   16.0837    1242880  13. Valet\PhpFpm->install() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/PhpFpm.php:178
   22.7151    1242880  14. Valet\Pecl->installExtensions() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/PhpFpm.php:75
   22.8074    1242880  15. Valet\Pecl->installExtension() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php:92
   23.0048    1242880  16. Valet\Pecl->install() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php:113
   48.5202    1288128  17. Valet\Filesystem->get() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php:142
   48.5202    1288128  18. file_get_contents() /Users/nmak/.composer/vendor/weprovide/valet-plus/cli/Valet/Filesystem.php:112


In Pecl.php line 144:

  Could not find ini definition for: apcu_bc in PHP Warning:  Module 'apcu' already loaded in Unknown on line 0Warning: Module
  'apcu' already loaded in Unknown on line 0/usr/local/etc/valet-php/7.3/php.ini


use [<phpVersion>]
```

**Comment by ashtonlance** [comment]
Same problem here.

Dirty fix: go to` /usr/local/etc/valet-php/7.3/php.ini` and remove the duplicate apcu load.

**Comment by samgranger** [comment]
Closing this issue since @ashtonlance provided a way to fix this issue. We will investigate how to possibly prevent this issue from occurring in the future.

**Comment by Judx** [comment]
Had the same error as @kupoback when trying to install valet with maria-db: `valet install --with-mariadb`.

@ashtonlance 's answer fixed the problem. Thank you!

**Comment by igor-imaginemage** [comment]
@ashtonlance answer temporarily fix the problem, but if enable xdebug for example the problem comes back.

To fix that I ran `sudo find / -name php.ini` to find all the php.ini that I had on my machine. So I delete all the php installations where these files were stored.

After that I ran `valet fix` and valet was able to reinstall php and make it work without any problem.

I hope this can also help you guys.
